webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/core_menu_options.test.tsx

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {DateTime} from 'luxon';
+import React from 'react';
+
+import {renderWithContext, fireEvent, screen} from 'tests/react_testing_utils';
+
+import CoreMenuOptions from './core_menu_options';
+
+jest.mock('components/advanced_text_editor/use_post_box_indicator', () => ({
+    __esModule: true,
+    default: jest.fn(),
+}));
+
+jest.mock('components/menu', () => ({
```

Comment:
I'm going to file a ticket for myself to make it so that this isn't needed. `Menu.Item` won't call its click handler when it's outside of a `Menu` due to a change I made previously, and I think I can address that as part of the making testing easier during the Hackathon tommorrow

Reply:
https://mattermost.atlassian.net/browse/MM-61560